### PR TITLE
16-11-2022

### PR DESCRIPTION
**Pull request by GOWTHAMBIKA**
image

### CODE REVIEW

File: cypress/e2e/components/test.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+export class Account{
```
Comment (qualybittech) [inline cypress/e2e/components/test.js:1]: We don't need to commit this file as code is not completed fully

File: cypress/e2e/test.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { Account } from "./components/accounts";
```
Comment (qualybittech) [inline cypress/e2e/test.js:1]: We don't need to commit this file as code is not completed fully

**Review by qualybittech — CHANGES_REQUESTED** [review]

File: cypress/e2e/test.js (hunk 2 above)
Comment (GOWTHAMBIKA) [reply cypress/e2e/test.js:1]: Ok .

**Comment by GOWTHAMBIKA** [comment]
Can merge this request ?

**Pull request closed without merging**